modules/fedimint-ln-server/src/lib.rs

```diff
@@ -946,8 +946,13 @@ impl Lightning {
         our_peer_id: PeerId,
         task_group: &TaskGroup,
     ) -> anyhow::Result<Self> {
+        let (block_count_tx, block_count_rx) = watch::channel(None);
         let btc_rpc = create_bitcoind(&cfg.local.bitcoin_rpc)?;
-        let block_count_rx = btc_rpc.spawn_block_count_update_task(task_group)?;
+        btc_rpc.spawn_block_count_update_task(task_group, {
+            move |count| {
+                let _ = block_count_tx.send(Some(count));
+            }
+        })?;
```

Comment:
Doesn't this require a notify?

Reply:
Spoiler: it did, but it was fixed in a followup commit already.